### DISCUSSION TITLE
Unbottled - how to undo

### DISCUSSION THREAD

**Discussion by rafrafek** [Homebrew/discussions#3652] [Everyday usage] [answered]
Hello,

I've accidentally ran command `brew unbottled`, and I wonder how can I fully undo it? It took a while to process unbottling and I want to undo all these operations. I don't want having anything unbottled on my computer.

**Comment by carlocab** [comment, marked as answer]
`brew unbottled` just lists information. There is nothing to undo.
```
❯ brew help unbottled
Usage: brew unbottled [options] [formula ...]

Show the unbottled dependents of formulae.
```
